src/script/interpreter.cpp

```diff
@@ -191,7 +191,7 @@ bool static CheckSignatureEncoding(const valtype &vchSig, unsigned int flags, Sc
     if (vchSig.size() == 0) {
         return true;
     }
-    if ((flags & (SCRIPT_VERIFY_DERSIG | SCRIPT_VERIFY_LOW_S | SCRIPT_VERIFY_STRICTENC)) != 0 && !IsValidSignatureEncoding(vchSig)) {
```

Comment:
No, the left-hand-side of `&&`` is true if any of these flags are set. Since DERSIG is always set, we don't need to consider the other flags any more.